packages/jira-adapter/test/change_validators/workflowsV2/referenced_workflow_deletion.test.ts

```
@@ -0,0 +1,112 @@
+ * limitations under the License.
+ */
+import { buildElementsSourceFromElements } from '@salto-io/adapter-utils'
+import {
+  toChange,
+  InstanceElement,
+  ReferenceExpression,
+  ReadOnlyElementsSource,
+  SeverityLevel,
+} from '@salto-io/adapter-api'
+import { referencedWorkflowDeletionChangeValidator } from '../../../src/change_validators/workflowsV2/referenced_workflow_deletion'
+import { WORKFLOW_SCHEME_TYPE_NAME } from '../../../src/constants'
+import { createEmptyType, createSkeletonWorkflowV2Instance } from '../../utils'
+
+describe('referencedWorkflowDeletionChangeValidator', () => {
+  const workflowSchemeObjectType = createEmptyType(WORKFLOW_SCHEME_TYPE_NAME)
+  let defaultSchemeInstance: InstanceElement
+  let referencingSchemeInstance: InstanceElement
+  let referencedWorkflowInstance: InstanceElement
+  let unreferencedWorkflowInstance: InstanceElement
+  let defaultWorkflowInstance: InstanceElement
+  let elementSource: ReadOnlyElementsSource
+
+  beforeEach(() => {
+    defaultWorkflowInstance = createSkeletonWorkflowV2Instance('defaultWorkflowInstance')
+    referencedWorkflowInstance = createSkeletonWorkflowV2Instance('referencedWorkflowInstance')
+    unreferencedWorkflowInstance = createSkeletonWorkflowV2Instance('unreferencedWorkflowInstance')
+
+    defaultSchemeInstance = new InstanceElement('defaultSchemeInstance', workflowSchemeObjectType, {
+      name: 'defaultSchemeInstance',
+      defaultWorkflow: new ReferenceExpression(defaultWorkflowInstance.elemID),
+      items: [],
+    })
+    referencingSchemeInstance = new InstanceElement('referencingSchemeInstance', workflowSchemeObjectType, {
+      defaultWorkflow: new ReferenceExpression(defaultWorkflowInstance.elemID),
+      name: 'referencingSchemeInstance',
+      items: [{ workflow: new ReferenceExpression(referencedWorkflowInstance.elemID) }],
+    })
+  })
+
+  it('should succeed because the deleted workflow is unreferenced.', async () => {
```

nit: IMO the phrases 'succeed' and 'fail' are unclear in this context, maybe `should return an error` and `should not return an error` are clearer

---

Done